### PR TITLE
Bug fix 123 & 128

### PR DESCRIPTION
**Pull request by aza547**
#123 
#128

### CODE REVIEW

**Review by tdaugaard — APPROVED** [review]
Looks fine to me!

File: src/main/recorder.ts [hunk 1]
```diff
@@ -130,6 +134,8 @@ const glob = require('glob');
 
         console.log(addColor("[Recorder] Stop recording buffer", "cyan"));
         clearInterval(this._bufferRestartIntervalID);
+        this._bufferRestartIntervalID = undefined;
```
Comment (ericlytle) [inline src/main/recorder.ts:137]: i know why you are doing this, but this really seems odd to me and kind of a hacky way to accomplish what we are wanting. 
can we not rely on the outcome of clearInterval and use that?

if we cant, do we really need clearInterval call then if we are setting the value to undefined immediately following. 

all these comments are based on my 2 min google search for what clearInterval does. so please correct me if im way off

Comment (tdaugaard) [reply src/main/recorder.ts:137]: It's not actuallt necessary to do this. 
image

Comment (tdaugaard) [reply src/main/recorder.ts:137]: It's not necessary to clear the timer ID, no; it references nothing when it's been cleared, even if it still has a value.

Comment (aza547) [reply src/main/recorder.ts:137]: Thanks. Agree I can just delete this line.

Comment (aza547) [reply src/main/recorder.ts:137]: Hm actually I'm less sure on second thought. Are you suggesting I do:

`this._bufferRestartIntervalID = clearInterval(this._bufferRestartIntervalID);` 

I'm relying on this for the guard elsewhere

Comment (tdaugaard) [reply src/main/recorder.ts:137]: I saw that you're testing it later for not being undefined. I wouldn't personally use a timer variable as a guard . I think I've done something similar elsewhere, where after clearing the interval I clear the guard variable.

I guess you _could_ double-use it by setting it to undefined - it doesn't hurt in that case.

Comment (aza547) [reply src/main/recorder.ts:137]: OK, going to keep it as it was initially then. Prefer that I think than having another variable.

**Pull request merged**